### PR TITLE
Addition of Photon HoE variable needed for H(gg) analysis

### PR DESCRIPTION
**Pull request by TizianoBevilacqua**
PR description:

With this pull request I added the two versions of Photon HoE calculation to the nAOD variable output. Previously for run 3 the HoE dumped was calo based while for run 2 it was Tower based. Both are needed in both cases for H(gg) analyses.

PR validation:

To test I've created a nAOD starting from Run2 data samples and checked that both version of HoE are dumped correctly

@lfinco @fcouderc

### CODE REVIEW

File: PhysicsTools/NanoAOD/python/photons_cff.py [hunk 1]
```diff
@@ -228,6 +228,7 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         pfRelIso03_chg_quadratic = Var("userFloat('PFIsoChgQuadratic')/pt",float,doc="PF relative isolation dR=0.3, charged hadron component (with quadraticEA*rho*rho + linearEA*rho Winter22V1 corrections)"),
         pfRelIso03_all_quadratic = Var("userFloat('PFIsoAllQuadratic')/pt",float,doc="PF relative isolation dR=0.3, total (with quadraticEA*rho*rho + linearEA*rho Winter22V1 corrections)"),
         hoe = Var("hadronicOverEm()",float,doc="H over E",precision=8),
+        hoe_TowOverEm = Var("hadTowOverEm()",float,doc="H over E Tower based calculation",precision=8),
```
Comment (hqucms) [inline PhysicsTools/NanoAOD/python/photons_cff.py:231]: @TizianoBevilacqua I found the name `hoe_TowOverEm` a bit confusing as `oe`:=`OverEm`. Would it be better to just call it `hadTowOverEm` or `hoe_Tower`?

**Comment by hqucms** [comment]
@TizianoBevilacqua I just realized that you opened this pull request directly to 14_0_X. Please first open one for the master branch.

**Comment by hqucms** [comment]
enable nano

**Comment by hqucms** [comment]
please test

**Comment by hqucms** [comment]
+1

**Comment by antoniovilela** [comment]
+1

**Comment by RSalvatico** [comment]
type egamma

**Pull request merged**